ui/webui/src/components/animations-workaround.css

```diff
@@ -0,0 +1,33 @@
+/*
+ * Animations in cockpit-client / webkit in VMs are causing excessive CPU load.
+ * See rhbz#1204242.
+ * Workaround it by decreasing the frame rate.
+ */
+
+.pf-c-skeleton {
+  --pf-c-skeleton--after--AnimationTimingFunction: steps(4);
```

Comment:
Have you tried 24 steps as in https://github.com/rhinstaller/anaconda/commit/eead6838d69699a7f329305c119857301eab5122? It is that magical threshold when the human eye stops to see pictures and starts to see an animation. It is a kind of a standard for animations.

Reply:
It was eating still to much CPU. I'll bring some numbers.

Reply:
VM with 8 GB RAM, 4 CPUs:
```
              WebKitWebProcess       gnome-kiosk
original           160 (peak 210)       60
steps(4)            30                  15
steps(24)          115 (peak 130)       50
```

Reply:
Interesting. Anyway, thanks for the data!